### DISCUSSION TITLE
How to get a PR reviewed?

### DISCUSSION THREAD

**Discussion by bame-da** [executablebooks/meta#926] [Q&A]
Hey all,

Firstly thanks for the work you are doing. We've been using a few of the plugins from the executablebooks ecosystem for our own docs. I recently discovered a small bug with the external-toc plugin and thought I'd give back and take a shot at fixing it myself rather than just opening an issue: https://github.com/executablebooks/sphinx-external-toc/pull/88

I know it's just been two days, but unlike with my initial push, the CI is now not running anymore and I've not had any human interaction on the PR. Is there anything in particular I should do to get some eyes on it for review?

**Comment by choldgraf** [comment]
Hey @bame-da - the short answer is that it often takes more than 2 days to review PRs. This project has a _lot_ of maintenance burden across many repositories, and relatively few people to help keep up with things, triage and respond to issues, review pull requests, etc. We are working on ways to open up team processes and grow a community around these projects, but it may take some time with the little resources that we have. Sorry that it is sometimes frustratingly slow :-/  

I know that @rowanc1 has been leading some monthly community meetings, so that might be one place to connect with others.
